src/main/ArrayVisualizer.java

```diff
@@ -86,6 +110,38 @@ of this software and associated documentation files (the "Software"), to deal
 final public class ArrayVisualizer {
     private static ArrayVisualizer INSTANCE = null;
 
+    private static final int STAT_LINE_BREAK    = 0;
+    private static final int STAT_SORT_IDENTITY = 1;
+    private static final int STAT_ARRAY_LENGTH  = 2;
+    private static final int STAT_FRAMERATE     = 13;
```

Comment:
why's framerate arbitrarily in the middle?

Reply:
Because I felt like this was the best place for it, and I already was using 0-12.

Reply:
but in the text file you comment it at the beginning, and why not just change the numbers?

Reply:
Moving it would take unnecessary effort, but the numbers could technically be changed.

Reply:
why would moving it take unnecessary effort?